docs/references/sdk-metrics.md

```diff
@@ -28,10 +28,10 @@ Each metric may have some combination of the following keys attached to them:
   - `workflow_task`
   - `activity_task`
   - `sticky_workflow_task`
-- `worker_type`: One of the following:
-  - `ActivityWorker`
+- `worker_type`: One of the below values. Note that if a single Worker is configured to handle all three types of tasks, it will emit all three metrics.
```

Comment:
I generally try to avoid "Note that", but didn't think of a better way (parens?) to frame this kind of side-note information inside this bullet. (But do think it belongs in the bullet.)

Reply:
I agree and can't think of a cleaner way to include it.